packages/html-reporter/src/templates/customElements.ts

```diff
@@ -0,0 +1,8 @@
+interface MutationTestReportAppHTMLElement {
+  titlePostfix: string;
+}
+declare namespace JSX {
+  interface IntrinsicElements {
```

Comment:
Where is this used?

Reply:
It is how to add custom elements to typed-html: https://www.npmjs.com/package/typed-html#custom-elements